src/bp/core/sdk/impl.ts

```diff
@@ -68,6 +69,10 @@ export class IOEvent implements sdk.IO.Event {
       this.credentials = args.credentials
     }
 
+    if (this.direction === 'outgoing') {
+      this.incomingEventId = args.incomingEventId ? args.incomingEventId : undefined
```

Comment:
The part `? args.incomingEventId : undefined` is redundant, if the argument isn't set, it will be undefined